src/index.html

```diff
@@ -2,12 +2,231 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <meta name="viewport"
+          content="width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0">
+    <title>Uber Eats</title>
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Hello Mate Academy</h1>
+    <header>
+      <div class="header">
+        <div class="container header__container">
+          <a id="page-top" href="#" class="header__logo logo_link"><img src="./images/logo.svg" alt="uber eats"></a>
```

Comment:
Делай переносы слишком длинных строк
```suggestion
          <a id="page-top" href="#" class="header__logo logo_link">
            <img src="./images/logo.svg" alt="uber eats">
          </a>
```